configs/tfm_mbedcrypto_config_profile_medium.h

```diff
@@ -8,10 +8,8 @@
  *  memory footprint.
  */
 /*
- *  Copyright (C) 2006-2022, ARM Limited, All Rights Reserved
+ *  Copyright The Mbed TLS Contributors
```

Comment:
`tfm_mbedcrypto_config_profile_medium.h` and `crypto_config_profile_medium.h` are imported from TF-M. I thought we wanted to keep them unmodified to facilitate updates. But it looks like this ship has already sailed, so never mind.

Reply:
Pros and cons either way. I think having a uniform license and copyright is probably a bigger win than having these files be identical across the two projects.